config/default/core.entity_view_display.block_content.hs_basic_block.default.yml

```diff
@@ -24,3 +24,4 @@ content:
     region: content
 hidden:
   field_hs_block_hide_title: true
+  search_api_excerpt: true
```

Comment:
Man, I hate that contrib modules do this. 

This does not seem related to this PR. Are you sure you want to have this here?

Reply:
yeah kinda annoying. it doesnt matter much. the search_api is only indexing nodes anyways. 
this is one of those things that i wanna try to figure out a method to update configs after a module is updated.